samples/ingestion/ingestion-client/FetchTranscription/TranscriptionProcessor.cs

```diff
@@ -155,8 +158,8 @@ private static async Task ProcessFailedTranscriptionAsync(string transcriptionLo
                         RetryCount = audio.RetryCount + 1
                     };
 
-                    var audioFileMessage = new Message(Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(sbMessage)));
-                    await ServiceBusUtilities.SendServiceBusMessageAsync(StartQueueClientInstance, audioFileMessage, log, TimeSpan.FromMinutes(1)).ConfigureAwait(false);
+                    var audioFileMessage = new Azure.Messaging.ServiceBus.ServiceBusMessage(JsonConvert.SerializeObject(sbMessage));
```

Comment:
```suggestion
                    var audioFileMessage = new ServiceBusMessage(JsonConvert.SerializeObject(sbMessage));
```

Reply:
@JoshLove-msft, There are ServiceBusMessage class in both Connector and Azure.Messaging.ServiceBus. So I need to distinguish them.